include/cantera/kinetics/RxnRates.h

```diff
@@ -91,6 +91,113 @@ class Arrhenius
     doublereal m_logA, m_b, m_E, m_A;
 };
 
+//! Blowers Masel reaction rate type depends on enthalpy 
+/**
+ * The Blowers Masel approximation is written by Paul Blowers,
+ * Rich Masel(DOI: https://doi.org/10.1002/aic.690461015) to 
+ * adjust the activation energy based on enthalpy change of a reaction:
+ * 
+ *   \f{eqnarray*}{
+ *        Ea &=& 0\; \text{if }\Delta H < -4E_0 \\ 
+ *        Ea &=& \Delta H\; \text{if }\Delta H > 4E_0 \\
+ *        Ea &=& \frac{(w + \Delta H / 2)(V_P - 2w + 
+ *               \Delta H)^2}{(V_P^2 - 4w^2 + (\Delta H)^2)}\; \text{Otherwise}
+ *   \f}
+ * where
+ *   \f[
+ *        V_P = \frac{2w (w_0 + E_0)}{w - E_0},
+ *   \f]
+ * \f$ w \f$ is the average of the bond energy of the 
+ * bond breaking and that being formed. Since the expression is 
+ * very insensitive to \f$ w \f$ for \f$ w >= 2 E_0 \f$, it 
+ * can be approximated to an arbitrary high value like \f$ 1000~kJ/mol \f$.
```

Comment:
The meaning of statement isn't entirely clear to me - what is "it" that can be approximated with a high value of _w_?

Also, units shouldn't be italicized, so no reason to put them in math mode.
```suggestion
  * can be approximated to an arbitrary high value like 1000 kJ/mol.
```